.github/workflows/yjit-ubuntu.yml

```
@@ -195,6 +195,7 @@ jobs:
           test -n "${LAUNCHABLE_STDOUT}" && exec 1> >(tee "${LAUNCHABLE_STDOUT}")
           test -n "${LAUNCHABLE_STDERR}" && exec 2> >(tee "${LAUNCHABLE_STDERR}")
 
+          verbose () { set -x && "$@" && set +x; } && verbose \
```

Can we just use `set -x`?

```suggestion
          set -x
```

I would rather want `zjit-*.yml` to use `|` instead of `>-` and call `set -x` there too.